agent/src/agent.php

```diff
@@ -108,12 +113,13 @@
 $ingestBrowser = $browserFactory(
     connectionTimeout: $ingestConnectionTimeout,
     timeout: $ingestTimeout,
-    server: $server,
     headers: [
+        'accept' => 'application/json',
         'content-encoding' => 'gzip',
-        'content-type' => 'application/octet-stream',
+        'content-type' => 'application/json',
```

Comment:
Although the request body is binary, we specify the `content-encoding` as GZIP.

This tells the server how decode the body from binary into its original format. This header should still indicate what the original body format was, in our case that is JSON.

<img width="811" alt="Screenshot 2025-04-11 at 13 24 39" src="https://github.com/user-attachments/assets/e84a7402-09c8-4668-8ac9-6458102992f1" />